### ISSUE TITLE
Build breaks on armv7: mismatched types

### ISSUE THREAD

**Issue by yurivict** [dalance/bsd-kvm#2]
```
error[E0308]: mismatched types
   --> /wrkdirs/usr/ports/sysutils/procs/work/procs-0.14.4/cargo-crates/bsd-kvm-0.1.3/src/lib.rs:344:19
    |
344 |             size: x.ki_size,
    |                   ^^^^^^^^^ expected `u64`, found `u32`

error[E0308]: mismatched types
   --> /wrkdirs/usr/ports/sysutils/procs/work/procs-0.14.4/cargo-crates/bsd-kvm-0.1.3/src/lib.rs:345:21
    |
345 |             rssize: x.ki_rssize,
    |                     ^^^^^^^^^^^ expected `i64`, found `i32`

error[E0308]: mismatched types
   --> /wrkdirs/usr/ports/sysutils/procs/work/procs-0.14.4/cargo-crates/bsd-kvm-0.1.3/src/lib.rs:346:20
    |
346 |             swrss: x.ki_swrss,
    |                    ^^^^^^^^^^ expected `i64`, found `i32`
```

Here is the [log](https://pkg-status.freebsd.org/ampere3/data/140releng-armv7-default/add1ba90ee22/logs/procs-0.14.4.log).

**Comment by dalance** [comment]
I've released v0.1.4.
Could you try it?

**Comment by yurivict** [comment]
Thanks for fixing it.

I've updated the FreeBSD port sysutils/procs, that was failing, to the latest revision that already contains 0.1.4 and will re-open this issue if there would still be failures on 32-bit architectures.
